Gordon360/Services/MembershipService.cs

```diff
@@ -444,5 +390,55 @@ public class ParticipationType
             // BUT, it's convenient to treat it as a participation type in several places throughout the API
             public static ParticipationType GroupAdmin { get { return new ParticipationType("GRP_ADMIN"); } }
         }
+
+        /// <summary>	
+        /// Converts a MembershipUploadViewModel object to a MEMBERSHIP
+        /// </summary>
+        /// <param name="membership">The MembershipUploadViewModel to convert</param>	
+        /// <returns>A new MEMBERSHIP object filled with the info from the view model</returns>	
+        public MEMBERSHIP MembershipUploadToMEMBERSHIP(MembershipUploadViewModel membership)
```

Comment:
Didn't you already move this to the ViewModel?

Reply:
Yes.